he_man_tenseal/inference.py

```diff
@@ -562,6 +563,9 @@ def execute(self, state: Dict[str, Union[ts.CKKSVector, np.ndarray]]) -> None:
     class AveragePoolOperator(GemmWrappedOperator):
         pass
 
+    class BatchNormalizationOperator(GemmWrappedOperator):
```

Comment:
This is okay for the moment. I will open a PR to combine the previous version (fast, but only for plaintext parameters) with this version.

Reply:
I will commit to this branch/PR